src/documentation/pages/components/date-picker/examples/_calendar-multiple-active-dates.vue

```diff
@@ -0,0 +1,78 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Multiple active dates" id="calendarr-multiple-active-dates" :tabs="exampleTabs">
+    chi-date(multiple, value="03/10/2021,03/11/2021,03/12/2021" slot="example")
+    <Wrapper slot="code-webcomponent">
+      .chi-tab__description Specify multiple active dates by separating the dates with a comma
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+      </pre>
+    </Wrapper>
+    <Wrapper slot="code-htmlblueprint">
+      .chi-tab__description
+        span.-text--grey Use class <code>-active</code> for the selected date
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      datePickerDates: Array.from(Array(30).keys()),
+      datePickerDayNames: ['S', 'M', 'T', 'W', 'T', 'F', 'S'],
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: `<chi-date value="03/10/2021,03/11/2021,03/12/2021" multiple></chi-date>`,
+        htmlblueprint: ``,
+      },
+    };
+  },
+})
+export default class CalendarMultipleActiveDates extends Vue {
```

Comment:
The HTML Blueprint code snippet is missing `<div class="chi-datepicker__day ">31</div>`